wdio.conf.js

```diff
@@ -73,10 +73,54 @@ exports.config = {
   // Sauce Labs platform configurator - a great tool to configure your capabilities:
   // https://docs.saucelabs.com/reference/platforms-configurator
   //
-  capabilities: {
+  // If CI && Safari run Safari + Edge
+  // If just Safari run Safari + Chrome
+  // IF not Safari run Firefox + Chrome
+  capabilities: process.env.SAFARI ? {
```

Comment:
Aren't you using the `--browsers` flag to run this here? Do we need to document the new env var?

Reply:
This is webdriver/wdio so I'm not entirely sure it does custom argument flags or that the config can accept them.
The widgets does a similar setup with using a ENV var to select Chrome/Firefox browsers